### PR TITLE
[SMAN-102] Create desktop gui for OnlineServicesEmulator

### LINKED ISSUE #3: [SMAN-102] Create desktop gui for OnlineServicesEmulator

### PR DESCRIPTION
**Pull request by radu1351**
CR URL: https://jira.harman.com/jira/browse/SMAN-102
CHANGES: Created the GUI
Testing: No testing required

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -17,14 +17,18 @@
     <dependency>
       <groupId>org.openjfx</groupId>
       <artifactId>javafx-controls</artifactId>
-      <version>17.0.6</version>
+      <version>20</version>
     </dependency>
```
Comment (sebi1995) [inline pom.xml]: format the code,
for however cute java 20 might feel and look, please restrict yourself to java 9
also only create the bare bones of the application, just something that will start up, a blank, the other features of the application are addressed in the other tickets

**Review by sebi1995 — CHANGES_REQUESTED** [review]

File: src/main/java/com/example/onlineservicesemulator/Application.java [hunk 2]
```diff
@@ -1,18 +1,16 @@
...
 
-import javafx.application.Application;
 import javafx.fxml.FXMLLoader;
 import javafx.scene.Scene;
 import javafx.stage.Stage;
 
 import java.io.IOException;
 
-public class HelloApplication extends Application {
+public class Application extends javafx.application.Application {
```
Comment (sebi1995) [inline src/main/java/com/example/onlineservicesemulator/Application.java]: rename application to OnlineServicesEmulator
and remove javafx.application.

File: pom.xml [hunk 3]
```diff
@@ -2,73 +2,127 @@
...
+            <groupId>org.junit.jupiter</groupId>
+            <artifactId>junit-jupiter-engine</artifactId>
+            <version>${junit.version}</version>
+            <scope>test</scope>
+        </dependency>
+    </dependencies>
 
-  <build>
+    <build>
         <plugins>
```
Comment (sebi1995) [inline pom.xml:48]: these plugins were for a different task

File: src/main/java/com/example/onlineservicesemulator/Controller.java [hunk 4]
```diff
@@ -0,0 +1,7 @@
+package com.example.onlineservicesemulator;
```
Comment (sebi1995) [inline src/main/java/com/example/onlineservicesemulator/Controller.java]: this class will be for a different task, delete it

**Review by sebi1995 — CHANGES_REQUESTED** [review]

**Review by AndreiK83 — APPROVED** [review]

**Comment by AndreiK83** [comment]
There are no major changes, so you can merge it to the main branch ;)

**Pull request merged**